src/sorespo/rfs.act

```diff
@@ -183,8 +188,124 @@ end-policy"""
         raise UnsupportedDevice()
 
 
+actor BBInterfaceTransform(update_dynstate: proc(?orchestron_rfs__rfs__backbone_interface__dynstate) -> None, dev: odev.DeviceMgr):
+    IF_STATE_INIT = "init"
+    IF_STATE_INTERFACE_MONITOR = "interface-monitor"
+    IF_STATE_INTERFACE_OK = "interface-ok"
+    IF_STATE_ISIS_OK = "isis-ok"
+    IF_STATE_ALL_OK = "all-ok"
+
+    rpc_root = crpd23.rpc_root(odev.DeviceTreeProvider(dev))
+    var interface_name = None
+    var healthcheck_enabled = False
+    var healthcheck_running = False
+    var health_state = IF_STATE_INIT
+    var interface_healthy = False
+    var isis_healthy = False
+    var ldp_healthy = False
+    var counters_recorded = False
+    var t0_rx_errors = None
+    var t0_rx_dropped = None
+    var t0_rx_over_errors = None
+    var t0_tx_errors = None
+    var t0_tx_dropped = None
+    var t0_tx_carrier_errors = None
```

Comment:
Store one tuple instead, not the individual fields. I reckon you could even store a little list of tuples for the last couple of polls.